### DISCUSSION TITLE
Account suspended and support ignores me

### DISCUSSION THREAD

**Discussion by alisherodilov2** [community/community#88543] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hello i had banned from git hub https://github.com/alisherodilov this account i did not get any warning i was testing my project this firstly i could not get access to my laptop and know  this warning appearing when i try to login in to my account 

Account suspended
Access to your account has been suspended due to a violation of our [Terms of Service](https://docs.github.com/articles/github-terms-of-service).

can someone help me with this 
Please [contact support](https://support.github.com/contact/reinstatement) for more information.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @alisherodilov005, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
